git-clang-format

```diff
@@ -144,10 +148,22 @@ def main():
   # The computed diff outputs absolute paths, so we must cd before accessing
   # those files.
   cd_to_toplevel()
+
+  # If told to, temporarily retrieve a .clang-format style file
+  to_delete = False
+  if opts.style_path:
+    to_delete = copy_format_file(opts.style_path)
+    if opts.verbose >= 1:
+      if already_exists:
```

Comment:
When running the script, I get `NameError: global name 'already_exists' is not defined`..